src/server/src/main/java/io/cassandrareaper/storage/CassandraStorage.java

```diff
@@ -248,6 +260,8 @@ private static void initializeAndUpgradeSchema(
         if (currentVersion <= 15) {
           Migration016.migrate(session, keyspace);
         }
+        // Switch metrics table to TWCS if possible
```

Comment:
`// Switch metrics table to TWCS if possible, this is intentionally executed every startup`